tests/testthat/test_vcov.R

```diff
@@ -0,0 +1,42 @@
+library(modmarg)
+context("Use different variance-covariance matrices")
+
+test_that("clustered standard errors are correct", {
+
+  data(margex)
```

Comment:
can you add a comment here that you're just comparing gaussian vs binomial for these two models? it just took me a while to figure out what's going on